app/routes/user.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+const bodyParser = require('body-parser');
```

Comment:
maybe we should import body parser in index.js so we are not importing it multiple times?